src/lib/stepper/_stepper-theme.scss

```diff
@@ -17,11 +17,15 @@
 
     .mat-step-label,
     .mat-step-optional {
-      color: mat-color($foreground, disabled-text);
+      // TODO(josephperrott): Update to using a corrected disabled-text contrast
+      // instead of secondary-text.
```

Comment:
Could you file an issue to track the bad contrast ratio on disabled text and reference it here?

Reply:
Create #14013